### PR TITLE
batch number 5 of mdn contribute docs fixes

### PR DESCRIPTION
**Pull request by chrisdavidmills**

### CODE REVIEW

File: files/en-us/mdn/tools/kumascript/index.html [hunk 1]
```diff
@@ -320,7 +320,7 @@ <h3 id="Using_search_keywords_to_open_template_pages">Using search keywords to o
 
 <p>The resulting dialog looks something like this:</p>
 
-<p><img alt="" src="https://mdn.mozillademos.org/files/14432/Screen%20Shot%202016-11-28%20at%203.08.39%20PM.png" style="height: 289px; width: 500px;"></p>
+<p><img alt="" src="screen_shot_2016-11-28_at_3.08.39_pm.png" style="height: 289px; width: 500px;"></p>
```
Comment (nschonni) [inline files/en-us/mdn/tools/kumascript/index.html:323]: Might be good to rename the file and add alt text

File: files/en-us/mdn/structures/page_types/index.html [hunk 2]
```diff
@@ -22,25 +22,14 @@
 
 <h2 id="Creating_a_new_page">Creating a new page</h2>
 
-<p>To create a new page on MDN, you first need "page creator" permissions. If you've not got this, you can request it from a member of the MDN admin team (use the <a href="mdn-admins@mozilla.org">mdn-admins mailing list</a>).  Once you've got this, you can create a new page in a couple of ways:</p>
-
-<ul>
- <li>On any page, you can create a new subpage using the <em>New sub-article</em> menu option in the gear menu.<img alt="options or gear menu on mdn, which includes the new sub-article option " src="https://mdn.mozillademos.org/files/15912/171EA6E3-FEA6-4B2D-AFF7-31E52A1FEA09.png" style="border-style: solid; border-width: 1px; display: block; height: 300px; margin: 0px auto; width: 233px;"></li>
```
Comment (nschonni) [inline files/en-us/mdn/structures/page_types/index.html:28]: Looks like the image got extracted, but isn't in the text anymore

File: files/en-us/mdn/tools/kumascript/index.html (hunk 1 above)
Comment (chrisdavidmills) [reply files/en-us/mdn/tools/kumascript/index.html]: Done, thanks!

File: files/en-us/mdn/structures/page_types/index.html (hunk 2 above)
Comment (chrisdavidmills) [reply files/en-us/mdn/structures/page_types/index.html:28]: thanks! Unused image deleted.

**Comment by pocketdr** [comment]
Excuse me, could you tell the reason of removing the description of obsolete_inline at /MDN/Structures/Macros/Commonly-used_macros?

**Comment by jpmedley** [comment]
We track this in [a separate database now](https://github.com/mdn/browser-compat-data) and display that data on the page.

**Comment by hamishwillee** [comment]
@pocketdr More precisely, MDN does not use "obsolete" any more either in the docs or the browser compatibility data. Instead the term "deprecated" is used. This should broadly be interpreted to mean "use of this API is discouraged" (i.e. it does not necessarily mean that the feature "is flagged for future removal" - it could be discouraged for lots of reasons).

**Comment by pocketdr** [comment]
> @pocketdr More precisely, MDN does not use "obsolete" any more either in the docs or the browser compatibility data. Instead the term "deprecated" is used. This should broadly be interpreted to mean "use of this API is discouraged" (i.e. it does not necessarily mean that the feature "is flagged for future removal" - it could be discouraged for lots of reasons).

I see. Thanks!

**Pull request merged**